### DISCUSSION TITLE
No 'Visualize Logic' for Presence

### DISCUSSION THREAD

**Discussion by TheDarkThief** [chakra-ui/zag#1444] [General]
# 🐛 Bug report
Attempting to 'Visualize Logic' for the Presence component pulls up an error 500 page

## 💥 Steps to reproduce

1. Go to ['Presence - Zag'](https://zagjs.com/components/react/presence)
2. Click on 'Visualize Logic'
3. See error

## 💻 Link to reproduction

[The empty page](https://state-machine-viz.vercel.app/presencel)

## 🧐 Expected behavior

A state machine diagram e.g. [Avatar Machine](https://state-machine-viz.vercel.app/avatar)

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      |   All       |
| Browser          |All|
| Operating System |     All      |

**Comment by segunadebayo** [comment]
I pushed a fix for this. Should be live shortly.

**Comment by segunadebayo** [comment]
Closing — the website and docs have been rebuilt for v1.0.
